mesonbuild/utils/universal.py

```diff
@@ -1091,6 +1092,16 @@ def has_path_sep(name: str, sep: str = '/\\') -> bool:
     return False
 
 
+def flatten_path(name: str, sep: str = '/\\', replace: str = '_') -> str:
+    'Replaces any specified @sep path separators in @name with @replace'
```

Comment:
This needs to be  triple quote to be a documentation string:
```
'''Repalces any  ...'''
```

Reply:
Interesting. The `has_path_sep` function just above in that file uses single quotes, and I was just following along :-)

Reply:
I don't think it matters... they are all just strings, and a throwaway string without assignment as the first thing in a module/class/function is a docstring.

But docstrings usually have complex English in them, so triple-quoted strings are crucial to make them well-formed in the first place rather than breaking all over apostrophes.

Reply:
Apparently it is just convention, but such a strong convention I was today years old when I learned it wasn’t a requirement